src/App.tsx

```diff
@@ -1,16 +1,41 @@
-import { Route, Routes } from "react-router-dom";
-import Body from "./components/Body";
+import { Route, Routes, useNavigate } from "react-router-dom";
+import { onAuthStateChanged } from "firebase/auth";
+import { useDispatch } from "react-redux";
+import { useEffect } from "react";
 import "./index.css";
```

Comment:
styles should be placed at the bottom (exception on multiple level nesting  but this has single nesting)

Reply:
updated